feature/gribi/ate_tests/hierarchical_weight_resolution_test/hierarchical_weight_resolution_test.go

```diff
@@ -256,8 +256,17 @@ func (a *attributes) configSubinterfaceDUT(t *testing.T, intf *oc.Interface) {
 // Sub Interfaces are also configured if numSubIntf > 0.
 func (a *attributes) configInterfaceDUT(t *testing.T, d *ondatra.DUTDevice, p *ondatra.Port) {
 	t.Helper()
-	i := a.NewOCInterface(p.Name())
-
+	i := &oc.Interface{Name: ygot.String(p.Name())}
```

Comment:
So this will still have GetOrCreateSubinterface(0) that is empty. If that's undesirable, feel free to rewrite this:

https://github.com/openconfig/featureprofiles/blob/f1c64ba6d85d17773ce595f490f14adbf07bd5d5/internal/attrs/attrs.go#L76-L103

To something like this:

```go
if a.IPv4 != "" {
  s := intf.GetOrCreateSubinterface(0)
  ...
}

if a.IPv6 != "" {
  s := intf.GetOrCreateSubinterface(0)
  ...
}
```

Reply:
> Looks like vlan 0 on port 2 is unused by this test. If you don't want to configure the untagged subinterface, just remove the IPv4 and IPv4Len above on lines 94-95 (also 104-105), and revert the change here.
> 
> Also update the README.md to be clear that we don't configure the untagged subinterface anymore?
> 
> FYI @Ankur19


[liulk](https://github.com/liulk), please check internal b/267822488 on the deviation for details. This is a requirement at the moment.

Reply:
I have an updated proposal there. Let's revisit this PR after we reach an agreement.

Reply:
Updated as per https://github.com/openconfig/featureprofiles/pull/1086.